src/status_im/contexts/wallet/add_account/create_account/view.cljs

```diff
@@ -57,125 +54,216 @@
+    :state}]
+  (let [derivation-path     (reagent/atom "")
+        set-derivation-path #(reset! derivation-path %)]
+    (fn [{:keys [customization-color]}]
+      (let [{:keys [derived-from
+                    key-uid]} (rf/sub [:wallet/selected-keypair])
+            on-auth-success   (rn/use-callback
+                               (fn [password]
+                                 (let [preferences {:account-name @account-name
+                                                    :color        @account-color
+                                                    :emoji        @emoji}]
+                                   (rf/dispatch
+                                    [:wallet/derive-address-and-add-account
+                                     {:password             password
+                                      :derived-from-address derived-from
+                                      :derivation-path      @derivation-path
+                                      :account-preferences  preferences}])))
+                               [derived-from])]
+        (rn/use-effect
+         #(rf/dispatch
+           [:wallet/next-derivation-path
+            {:on-success  set-derivation-path
+             :keypair-uid key-uid}])
+         [key-uid])
+
+        [floating-button
+         {:account-color      @account-color
+          :slide-button-props {:on-auth-success on-auth-success
+                               :disabled?       (or (empty? @account-name)
+                                                    (= "" @derivation-path))}}
+         [avatar
+          {:account-color   @account-color
+           :emoji           @emoji
+           :on-select-emoji set-emoji}]
+         [input
+          {:account-color  @account-color
+           :account-name   @account-name
+           :on-change-text on-change-text}]
+         [color-picker
+          {:account-color     @account-color
+           :set-account-color set-account-color}]
+         [new-account-origin
+          {:derivation-path     @derivation-path
+           :customization-color customization-color}]]))))
+
+(defn view
+  [_]
+  (let [account-name      (reagent/atom "")
+        account-color     (reagent/atom (rand-nth colors/account-colors))
+        emoji             (reagent/atom (emoji-picker.utils/random-emoji))
+        on-change-text    #(reset! account-name %)
+        set-account-color #(reset! account-color %)
+        set-emoji         #(reset! emoji %)
+        state             {:account-name  account-name
+                           :account-color account-color
+                           :emoji         emoji}]
+    (fn []
+      (let [customization-color    (rf/sub [:profile/customization-color])
+            ;; Having a keypair means the user is importing it or creating it.
+            {:keys [keypair-name]} (rf/sub [:wallet/create-account-new-keypair])]
+
+        (rn/use-unmount #(rf/dispatch [:wallet/clear-create-account]))
+
```

Comment:
Empty line

Reply:
They were left on purpose, to visually separate the `use-unmount` from the hiccup.  is this against the guidelines? :thinking:

Reply:
> is this against the guidelines?

I am not sure if we have guidelines for new lines, but I haven't seen that style in the codebase 

cc @ilmotta

Reply:
We don't have hard rules about blank lines. Zprint is configured to respect blank lines so we give some room for interpretation. It's all based on common sense right now. But as @OmarBasem said, we usually don't add a blank line between the final let form and the body of the component. On the other hand, the blank line after `use-unmount` I usually consider adding myself, but many times I forget.